### PR TITLE
Update orders

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the Manage Orders and Fulfillment section of ASC 4.0+ and some related topics.

## Affected documentation pages

- ASC 4.0+ > Manage Orders and Fulfillment
- new topic for Cancel Unshipped Orders.

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

### CODE REVIEW

File: src/sales-channels/asc/managing-orders.md [hunk 1]
```diff
@@ -1,71 +1,60 @@
...
 
-Amazon Sales Channel imports your Amazon Seller Central orders into Magento for review on the _Orders_ tab on the Amazon Sales Channel home page.
+Order information received from Amazon Seller Central shows in the _Recent Orders_ section of the [store dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}).
 
-Orders that are imported:
+Regardless of your [order import settings]({% link sales-channels/asc/order-settings.md %}), Amazon orders that existed in your Amazon Seller Central account prior to your [store integration]({% link sales-channels/asc/store-integration.md %}) will not import into Magento.
 
-- Orders that are in `Unshipped` status in your Amazon Seller Central account at the time of [store integration]({% link sales-channels/asc/store-integration.md %}) during onboarding.
-- All new orders in your Amazon Seller Central account after [store integration]({% link sales-channels/asc/store-integration.md %}).
+## Disabled Order Import
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-orders.md]: Should this be sentence case or is this a proper noun?

File: src/sales-channels/asc/managing-orders.md [hunk 2]
```diff
@@ -1,71 +1,60 @@
...
 
-- Orders that existed in your Amazon Seller Central account at the time of [store integration]({% link sales-channels/asc/store-integration.md %}) but were in a status other than `Unshipped`.
+If your [order import settings]({% link sales-channels/asc/order-settings.md %}) are set to `Disabled`, order information received from Amazon appears in the _Recent Orders_ section of the store dashboard. Order information is view only, and managing these orders must be done in Amazon Seller Central.
 
-An Amazon order is added to the to the _Orders_ tab when Amazon Sales Channel receives information that Amazon has changed the order status from `Pending` to `Unshipped` in Amazon Seller Central. Amazon places an order in `Pending` status while the order and the payment method are verified. Remember, information sync depends on your cron settings and the time it takes Amazon to process information.
+![]({% link sales-channels/asc/assets/amazon-recent-orders-not-imported.png %}){: .zoom}
+_Recent Orders - Not Imported_
 
-You can review your Amazon order information on the _Orders_ tab.
+## Enabled Order Import
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-orders.md]: Sam as other heading

File: src/sales-channels/asc/order-settings.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ Orders imported from Amazon are managed in the [Magento Orders]({% link sales/or
 {:.bs-callout-info}
 When an order is received from Amazon, your Amazon store is set to `Create Magento Order`, and the ordered item SKU cannot be matched to an existing Magento catalog product, the order synchronization process: <br/>- creates a Magento catalog product with the SKU<br/>- Sets the product status to `Disabled` and the inventory setting to `Do Not Manage Inventory`<br/>- Creates the Magento order
 
-# Configure Order Settings
+## Configure Order Settings
```
Comment (leslietilling) [inline src/sales-channels/asc/order-settings.md]:
```suggestion
## Configure order settings
```

File: src/sales-channels/asc/managing-orders.md (hunk 2 above)
Comment (hickskenh) [reply src/sales-channels/asc/managing-orders.md]: reworded.  denotes text that applies to each setting.

**Comment by hickskenh** [comment]
running tests

**Pull request merged**